src/test/java/seedu/address/logic/commands/EditPersonCommandTest.java

```diff
@@ -72,10 +74,11 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
 
     @Test
     public void execute_noFieldSpecifiedUnfilteredList_success() {
-        EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, new EditPersonDescriptor());
+        EditPersonCommand editCommand = new EditPersonCommand(INDEX_FIRST_PERSON, new EditPersonDescriptor());
```

Comment:
```suggestion
        EditPersonCommand editPersonCommand = new EditPersonCommand(INDEX_FIRST_PERSON, new EditPersonDescriptor());
```
Do you think it's worth renaming these variables as well? I know it doesn't change the functionality, but perhaps it's more explicit. Same for everywhere else in this file. Wdyt?